ols/src/tools/tools.py

```diff
@@ -69,7 +70,7 @@ async def _execute_single_tool_call(
         logger.error("Tool call missing name: %s", tool_call)
     else:
         try:
-            raise_for_sensitive_tool_args(tool_args)
+            # raise_for_sensitive_tool_args(tool_args)
```

Comment:
temporarily disabled `raise_for_sensitive_tool_args` because word `secret` is on the list and is used in Openshift MCP.